osu.Game/Overlays/Chat/ChatLine.cs

```diff
@@ -147,6 +144,8 @@ protected override void LoadComplete()
 
             updateMessageContent();
             FinishTransforms(true);
+
+            prefer24HourTime.BindValueChanged(_ => updateTimestamp(), true);
```

Comment:
```suggestion
            prefer24HourTime.BindValueChanged(_ => updateTimestamp());
```

Is already being run just above in `updateMessageContent()`